README.md

```diff
@@ -113,13 +107,13 @@ config, allowing only specific config to be controlled.
 ### Choosing config to export ###
 
 It is the developer's responsibility to be aware of what configuration should be
-exported to a module, but here is some help! Start by adding items you know you
+exported to a module (or profile), but here is some help! Start by adding items you know you
 want, such as `node.type.article`. Then run the following drush command, which
 will list anything that is dependent on the listed config, and all the
```

Comment:
Since this new command only actually lists things that are dependant on the listed config, please can we remove the 'and all the dependencies' bit here? I haven't read through other lines that haven't changed in the readme, but you might want to check there's nowhere else that needs updating similarly?